docs/working_with_page.md

```diff
@@ -0,0 +1,226 @@
+    |Gallery|Displays all images contained in a selected folder.|
+    |Video|Embeds a video into the Page with standard playback controls.|
+    |RSS|Loads and displays news from RSS feeds (channels).|
+    |Content Scheduler|Displays Content items at a pre-define time.|
+    |Text|Enables you to add to the Page a Rich Text block.|
+    
+    ## Block settings
+    
+    ### Blocks of content
+    
+    Each kind of block has its special properties. 
+    You can access them by placing the cursor on the added block and clicking the cog button.
+    Use the same menu to remove the block from the Page by clicking the trash button.
+    
+    ![Elements settings](img/page_builder_elementsettings.png)
+    
+    Settings available for blocks are divided into three tabs — **Basic**, **Design**, and **Scheduler**.
+    The settings available in the **Basic** tab are tailored to the block's content.
+    The remaining tabs contain parameters common to all blocks provided with eZ Enterprise. 
+    Use them to modify the layout and visibility of a block.
+    Always ensure that you set all the required settings of a block of content to successfully publish the Page.
+        
+    In the **Design** tab, perform the following actions:
+    
+    - From the **View** list, select the block layout.
+    - *Optional:* In the **Class** box, define the class of the block
+    - *Optional:* In the **Style** box, add additional style formatting.
+    
+    In the **Scheduler** tab, define the visibility time of the block by performing the following actions:
+    
+    - In the **Reveal** section, set the visibility to **Now** or plan it for future publish by clicking **Set date** and setting the date.
+    - In the **Hide** section, disable the option to hide the block by selecting **Never** or 
+    plan to hide it in future by clicking **Set date** and setting the date.
+    
+    !!! tip
+        Settings and function of custom-made blocks of content depend on their design. 
+        For details on adjusting and using them, contact your website administrator.
+    
+    ### Form block
+    
+    Note that completing the settings of the Form block requires at least one Form Content item created.
+    
+    In the **Basic** tab, perform the following actions:
+    
+    - In the **Name** box, provide the name of the block that will appear on the Page.
+    - In the Form section, click **Select content**, navigate through the content and select a Form Content item to append it to the block.
+    
+    Click **Submit** to save the changes or **Discard** to cancel.
+    
+    ### Code block
+    
+    In the **Basic** tab, perform the following actions:
+    
+    - In the **Name** box, provide the name of the block that will appear on the Page.
+    - In the **Content** box, enter the HTML code for the content you want to display on the Page.
+    You can start with the part of the code that generally belongs to the `body` block.
```

Comment:
Not sure about this sentence. Does it mean that you should not include `<head>` here?